charms/docker/docker.py

```diff
@@ -0,0 +1,87 @@
+import os
+import subprocess
+
+from shlex import split
+
+from .workspace import Workspace
+
+
+class Docker:
+    '''
+    Wrapper class to communicate with the Docker daemon on behalf of
+    a charmer. Provides stateless operations of a running docker daemon
+    '''
+
+    def __init__(self, socket="unix:///var/run/docker.sock", workspace=None):
```

Comment:
Done, and done. Thanks for pointing that out, i completely missed it even after you called out the funky one that was indeed correct. #whatamidoingwithmylife